### DISCUSSION TITLE
DOSXYZnrc(define phantom)

### DISCUSSION THREAD

**Discussion by Mona99-del** [nrc-cnrc/EGSnrc#1173] [General]
![image](https://github.com/user-attachments/assets/90d3f124-55d7-484b-b6e8-94e6d535c607)
After building the accelerator using BEAMnrc, I shared the library for the accelerator. Now, when starting to use dosxyznrc, I chose source 9. Then I found the phantom, as shown in the picture, to find a water phantom with dimensions of 10*30*30 cm3. I want to see the dose profile at the center. Do the steps in the picture give me what I want after running?
![image](https://github.com/user-attachments/assets/3d40320d-7e3b-4c3d-b66a-71167e1bab3a)

**Comment by rtownson** [comment]
You have the minimum x and y boundaries at 0, but your beam is also along 0. So while you are getting the z profile along this axis, your phantom is not centered. You probably want to set those minimum boundaries to `-15cm`. Also, I don't remember the beam angles off hand for a beam along z.

Consider checking the box to generate a .egsphant file (it's in the phantom settings). Then use another tool like VICTORIA to view the egsphant and 3ddose files that are produced by the simulation. This will ensure the beam angle is what you expect.

**Comment by Mona99-del** [comment]
![image](https://github.com/user-attachments/assets/13c213ec-168b-4bf2-9994-28e694088169)
Thank you for your reply. I was trying to use the feature in the image to calculate a specific voxel, but it seems I understood it and made the wrong selection. Can you explain it to me  Saving Time in Depth-Dose and Dose Profile Calculations using dflag and dsurround?

**Reply by rtownson** [reply]
The idea there is just to create a single column of voxels along Z (so the number of voxels in x and y is 1). Then you can use the "region surrounding phantom" parameters to fill out with the same material in the x and y directions. Though for the relatively small number of voxels you're simulating to start with, this seems barely necessary. If you want to do high resolution profiles, then it can be helpful (though it only extracts one at a time!).

**Comment by Mona99-del** [comment]
Ok, thanks for the clarification.
